tasks/align-and-call.wdl

```diff
@@ -0,0 +1,545 @@
+version 1.0
+
+import "https://raw.githubusercontent.com/gatk-workflows/gatk4-mitochondria-pipeline/1.0.0/tasks/alignment-pipeline.wdl" as AlignAndMarkDuplicates
```

Comment:
Not sure how you handle the import versions. Should this be 1.1.0? Just let me know what version number this should be.